docs/canvas/canvas-workpad.asciidoc

```diff
@@ -1,72 +1,304 @@
+. Apply a set of styles to the entire workpad.
 
 .. Expand *Global CCS overrides*.
 
 .. Enter the CCS, then click *Apply stylesheet*.
 
 [float]
-=== Share your workpad
+[[element-intro]]
+== Showcase your data with elements
 
-When you are ready to share your workpad, create a PDF, or export your workpad to another author.
+Elements are the building blocks of your workpad. You can add and combine elements to create dynamic displays of your data.
 
 [float]
-==== Create a PDF of your workpad
+[[add-canvas-element]]
+=== Add elements
+
+Choose the elements that best display your brand and data.
+
+. Click *Add element*.
+
+. On the *Elements* window, select the element you want to use. 
++
+[role="screenshot"]
+image::images/canvas-element-select.gif[Canvas elements]
+
+. Elements come preconfigured with a small data set so that you can familiarize yourself with the element. Play around with the default settings and see what the element can do. 
+
+TIP: Want to use a different element? You can delete the element by selecting it, clicking the *Element options* icon in the top right corner, then selecting *Delete*.
+
+[float]
+[[connect-element-data]]
+=== Connect the element to your data
+
+When you are ready to move on from the demo data, connect the element to your own data.
+
+. Make sure that the element is selected, then select *Data*.
+
+. Click *Change your data source*.
+
+[float]
+[[elasticsearch-sql-data-source]]
+==== Connect to Elasticsearch SQL
+
+Access your data in Elasticsearch using the Elasticsearch SQL syntax.
+
+. Click *Elasticsearch SQL*.
+
+. In the *Elasticearch SQL query* box, enter your query, then click *Preview*.
+
+. If everything looks correct, click *Save*. 
+
+Unfamiliar with writing Elasticsearch SQL queries? For more information, refer to {ref}/sql-spec.html[SQL language].
+
+[float]
+[[elasticsearch-raw-doc-data-source]]
+==== Connect to Elasticsearch raw data
+
+Use the Lucene query syntax to use your raw data in Elasticsearch.
+
+. Click *Elasticsearch raw documents*.
+
+. From the *Index* dropdown, select the index data that you want to display. 
+
+. From the *Fields* dropdown, select the associated fields you want to display.
 
-To view your workpad outside of Kibana, generate a PDF. 
+. To sort the data, select an option from the *Sort Field* and *Sort Order* dropdowns.
 
-. If you are using a basic license, enable reporting in your `config/kibana.yml` file.
+. For more targeted queries, enter a *Query* using the Lucene query string syntax. 
 
-. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+. Click *Preview*. 
 
-. Click *Generate PDF*. 
+. If your query looks correct, click *Save*. 
+
+For for more information about the Lucene query string sytax, refer to <<lucene-query,Lucene Query Syntax>>.
+
+[float]
+[[timelion-data-source]]
+==== Connect to Timelion
```

Comment:
I believe Timelion can now be disabled, should we mention something about that?

Reply:
@timductive could you provide some more information? If disabling Timelion is actionable from Canvas, I would say let's talk about it here.

Reply:
Nevermind, it looks like this doesn't affect the Canvas data source.